agoric/contract/src/proposal/start-kread-proposal.js

```diff
@@ -307,6 +327,10 @@ export const startKread = async (powers, config) => {
     brands: { KREAdCHARACTER: characterBrand, KREAdITEM: itemBrand },
   } = await E(zoe).getTerms(instance);
 
+  trace('adding to boardAux');
+  publishBrandInfo(chainStorage, board, characterBrand);
```

Comment:
Let's add an `await` to be sure `publishBrandInfo` has compelted

Reply:
![image](https://github.com/Kryha/KREAd/assets/5808153/f573d8ab-b784-411b-abc5-ff56e34e6765)